### ISSUE TITLE
custom icon

### ISSUE THREAD

**Issue by thestinger** [GrapheneOS/chromium_patches#9]

**Comment by csagan5** [comment]
See also: https://github.com/bromite/bromite/issues/120#issuecomment-418990771

Contains a link to https://opensourcedesign.net/jobs

**Comment by thestinger** [comment]
I ended up using an icon recolor for the time being:

https://github.com/GrapheneOS/chromium_patches/blob/pie/0020-Vanadium-branding-for-GrapheneOS.patch

I auto-converted the existing branding to a dark grayscale variant as I did previously. I still do want a custom icon, but I see some benefit in having it based on the Chromium logo to some extent since most people are familiar with that and can tell it's a browser from the icon, consciously or subconsciously. It's probably not ever going to be a standalone project from GrapheneOS, so it doesn't really need distinctive branding itself. It may make sense to make it more different from Chromium than just a recolor though.

I do have some progress on branding for GrapheneOS itself. I uploaded 3 PNGs generated from the initial set of vector graphics below (there are other assorted options to choose from too). I haven't entirely decided on the highlight color and I'll be making the theme in the OS and Chromium match this too once that's decided. I'd like to have the site match too. Currently planning on mostly sticking to matching the Material Light theme, but there are some colors that need to be chosen, likely from the Material color palette to match other things in the OS and apps. Also currently using the Roboto font for branding and the site for consistency, and a nice thing about that is it's very tied to AOSP. Google uses it for AOSP documentation, blog posts, etc. but mostly not for anything else tied to Google. They even replaced it in their own apps for titles / headers with Google Sans.

![aquaxetine graphene os png 19](https://user-images.githubusercontent.com/1505226/57584522-78b68280-74aa-11e9-8445-3ca4b32eb1fe.png)
![aquaxetine graphene os png 9](https://user-images.githubusercontent.com/1505226/57584524-7eac6380-74aa-11e9-862d-385059f22e7f.png)
![aquaxetine graphene os png 14](https://user-images.githubusercontent.com/1505226/57584519-6fc5b100-74aa-11e9-90e6-0028901830d5.png)
